oggm/core/gis.py

```diff
@@ -1958,6 +1963,7 @@ def reproject_gridded_data_variable_to_grid(gdir,
     if preserve_totals:
         # only preserve for float variables
         if not np.issubdtype(data, np.integer):
+            # do we want to do smoothing?
```

Comment:
```suggestion
            # TODO: do we want to do smoothing?
```
because is an open question and not a comment ;-) (also it is also a comment)